### ISSUE TITLE
let humans use telenodes as teleporters

### ISSUE THREAD

**Issue by Gireen** [Unvanquished/gameplay#5]
an old idea but it would give humans a reason to create forward bases and make them easier to defend.
https://forums.unvanquished.net/viewtopic.php?f=58&t=1906
With Unvanquished/Unvanquished#1232 and Unvanquished/Unvanquished#1216 it becomes possible to implement

**Comment by illwieckz** [comment]
or maybe a portal gun, but also usable by aliens while open 🤯

**Comment by DolceTriade** [comment]
It's a good idea, imo

**Comment by Viech** [comment]
I agree, no reason not to give humans more mobility in such a fun way.

**Comment by Viech** [comment]
@Gireen, it would be nice if you could flesh out your idea. How exactly would a player go about having them teleported? What are the limits?

**Comment by Gireen** [comment]
Stand on a telenode, press Q, select destination and teleport.
Teleporting would timeout the used telenodes similair to spawning.

**Comment by Viech** [comment]
How to select a destination? Experienced players tend to avoid UI usage as much as possible as it's too slow for the game. Maybe hitting the key could let you go to a randomly selected (or to the "next") different base (using base clusters)?

With respect to the cost, just timing out the telenode would make both intentional griefing and unintentional harmful play towards your teammates possible by preventing them from spawning back in. Maybe have it cost credits or health as well or instead? Do we want a short animation before the teleport happens?

**Comment by Gireen** [comment]
>How to select a destination?

with Unvanquished/Unvanquished#1232 

>Experienced players tend to avoid UI usage as much as possible as it's too slow for the game. Maybe hitting the key could let you go to a randomly selected (or to the "next") different base (using base clusters)?

yes. something like random, closest (available) to destination, previously used.
An simpler option is to just have the telenodes connected in a fixed order that players learn themself witch telenode leads where.
Ways for that could be something like build order or distance to the RC.

>With respect to the cost, just timing out the telenode would make both intentional griefing and unintentional harmful play towards your teammates possible by preventing them from spawning back in. Maybe have it cost credits or health as well or instead? 

I think for the beginning timeouts are enough. Witch also could affect the player, so that they can't grief. Other costs can be introduced if it would prove to powerful  

>Do we want a short animation before the teleport happens?

Sure. I mean its not necessary to have but added later together with an separate sound for teleportation.
